rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2022-02-14
+nightly-2022-05-01
```

Comment:
Why use 05-01?

Reply:
https://github.com/rust-lang/rust/issues/96132 is fixed in 04-17.